modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/rest/api/server/idp/v1/IdPSuccessTest.java

```diff
@@ -425,6 +431,106 @@ public void testAddIdP() throws IOException {
         assertNotNull(idPId);
     }
 
+    @Test()
+    public void addIdPWithoutAuthenticator() throws IOException {
+
+        String body = readResource("add-idp-without-authenticator.json");
+        Response response = getResponseOfPost(IDP_API_BASE_PATH, body);
+        response.then()
+                .log().ifValidationFails()
+                .assertThat()
+                .statusCode(HttpStatus.SC_CREATED)
+                .body("federatedAuthenticators.authenticators", Matchers.emptyIterable())
+                .header(HttpHeaders.LOCATION, notNullValue());
+
+        String location = response.getHeader(HttpHeaders.LOCATION);
+        assertNotNull(location);
+        String idpIdWithoutAuth = location.substring(location.lastIndexOf("/") + 1);
+        assertNotNull(idpIdWithoutAuth);
+
+        deleteCreatedIdP(idpIdWithoutAuth);
+    }
+
+
+    /* This test method has been added in order to test the current behaviour.
+     * There seem to be some concerns related to internal validations used in functionality associated with this.
+     * This is being tracked with the issue: https://github.com/wso2/product-is/issues/21928
+     */
+    @Test
+    public void addIdPWithDuplicatedOIDCScopes() throws IOException {
+
+        String oidcIdpPayload = readResource("add-oidc-idp.json");
+        String oidcScopesProperties = convertToJasonPayload(
+                createAuthenticatorProperties("Scopes","openid country profile"),
+                createAuthenticatorProperties("commonAuthQueryParams","scope=openid country profile"));
+//        String oidcScopesProperties = convertToJasonPayload(
```

Comment:
Remove this unwanted comments

Reply:
This is fixed with 2f657bedfa3cad42fd8c65f54e3b0befbc4119e0